mods/railcraft/api/tracks/TrackSpec.java

```diff
@@ -123,10 +123,10 @@ public ITrackInstance createInstanceFromSpec() {
         }
     }
 
-    public IIcon getItemIcon() {
+    public TextureAtlasSprite getItemSprite() {
         if (iconProvider == null)
```

Comment:
Probably. However I don't know how you want track block models to work so I have left it in there. It will be needed if you don't want to refactor the way that track models are generated.